snuba/migrations/groups.py

```diff
@@ -177,6 +179,16 @@ def __init__(
         storage_sets_keys={StorageSetKey.PROFILE_CHUNKS},
         readiness_state=ReadinessState.PARTIAL,
     ),
+    MigrationGroup.GENERIC_METRICS_V2: _MigrationGroup(
```

Comment:
I think it makes sense to have a storage_set per metric type. This gives us the ability to decide whether or we want 1-N clickhouse clusters for generic metrics. One implication I'd point out of this pattern (storage set per metric type in a single migration loader) is the inability to gradually roll out a new table in this migration group. For example, though maybe unlikely, say sentry decides we'd like to support a new metric type (RATE? HISTOGRAM?), if we create a new storage_set for this metric type and write the new table migration in this `GENERIC_METRICS_V2` loader, then it will automatically inherit its deployment state of this loader and go straight to production (SaaS, DE, ST, etc.). But to be clear, this is exactly how our current implementation of generic_metrics behaves today and we've managed this use-case with rolling out gauges in the past.

Reply:
That is a valid point. I had completely forgotten about this limitation of migrations. Will change the storage set to be different per metric type.